src/odemis/driver/lle.py

```diff
@@ -372,91 +371,78 @@ def _getIntensityGY(self, intensities):
 
         # Yellow has precedence over green
         if yellow_i is not None and intensities[yellow_i]: # don't use if None or 0
-            return intensities[yellow_i]
+            return intensities[yellow_i], yellow_i
         elif green_i is not None:
-            return intensities[green_i]
+            return intensities[green_i], green_i
         else:
-            return 0
+            return (0, 0)
```

Comment:
The second 0 is not really correct. It should still be one of the `yellow_i` or `green_i`. Of course, based on how it's used afterwards (multiply it by 0), that doesn't matter much. It'd just feel less odd to use a "random" channel.